pom.xml

```
@@ -32,7 +32,7 @@
   <properties>
     <deployable.classifier.wildfly>wildfly19</deployable.classifier.wildfly>
     <deployable.classifier.eap7>eap7</deployable.classifier.eap7>
-    <deployable.classifier.standalone>thorntail</deployable.classifier.standalone>       
+    <deployable.classifier.standalone>bootable</deployable.classifier.standalone>
```

could we take standalone classifier ?
```suggestion
    <deployable.classifier.standalone>standalone</deployable.classifier.standalone>
```